backend/app/admin_handler.go

```diff
@@ -251,7 +251,7 @@ func (h *Handler) GenerateVouchersHandler(c *gin.Context) {
 			models.WithSeverity(models.NotificationSeveritySuccess),
 			models.WithNoPersist(),
 		)
-		if err := h.notificationService.Send(c.Request.Context(), notif); err != nil {
+		if err := h.notificationService.Send(context.Background(), notif); err != nil {
```

Comment:
Can we use app ctx instead of context.Background (in all files) so on app exit all of these calls stop?